src/Settings.js

```diff
@@ -85,6 +85,66 @@ const ClearStorage = inject("store")(
   }
 );
 
+const FrequencyOptions = inject("store")(
+  observer(({ store }) => (
+    <div>
+      <h3>Frequency</h3>
```

Comment:
Round of phrasing suggestions (see also the following notes): "Tone Frequency (Hz)" here.